### DISCUSSION TITLE
tcpclientasync() class in Berry script is

### DISCUSSION THREAD

**Discussion by Bebuccio** [arendst/Tasmota#19039] [Q&A] [answered]
Using tasmota32 version 12.5.0 I have this  error using the berry instruction "var tcp = tcpclientasync()":

BRY: Exception> 'syntax_error' - input:2: 'tcpclientasync' undeclared (first use in this function)

There is an import directive to use tcp async client class?

**Comment by s-hadinger** [comment, marked as answer]
You need to upgrade to 13.0.0
